azure-pipelines.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           tensorflow: "tensorflow==1.13.1"
         Python37TF2:
           python.version: "3.7"
-          tensorflow: "tf-nightly-2.0-preview"
+          tensorflow: "tf-nightly-2.0-preview==2.0.0.dev20190603"
```

Comment:
Looks like the newest nightly has some build problems. I pinned it for now, so our builds are passing again.

Reply:
Tensorflow just released their [beta release](https://github.com/tensorflow/tensorflow/releases/tag/v2.0.0-beta0). I'll pin it to that.